integration_tests/integration/createLicence.spec.ts

```diff
@@ -31,12 +31,16 @@ context('Create a licence', () => {
 
     const appointmentTimePage = appointmentContactPage.enterTelephone('07892123456').clickContinue()
 
-    const additionalConditionsQuestionPage = appointmentTimePage
-      .enterDate(moment().add(1, 'year'))
+    /*
+     select the next wednesday one year from now in order to avoid weekends and
+     bank holidays, with the possible exceptions of Xmas and New Year
+     */
+    const additionalConditionsPage = appointmentTimePage
+      .enterDate(moment().add(1, 'year').add(1, 'week').day(3))
       .enterTime(moment())
```

Comment:
This will fail if Xmas, Boxing Day or NYD falls on a Weds - if the test runs in mid December! 
It might need to detect if the month is December, and if so, add a month? OK for now though, just ideas.

Reply:
Good idea! We can simply skip to January to avoid this issue. I'll push the change once complete.